command/server.go

```diff
@@ -1141,7 +1149,7 @@ func (c *ServerCommand) Run(args []string) int {
 				}),
 			})
 			var sealInfoKeys []string
-			var sealInfoMap = map[string]string{}
+			sealInfoMap := map[string]string{}
```

Comment:
I wonder why these were not covered by the big formatting PR that went in a few days back.

Reply:
This branch existed before the formatting PR. I think I just forgot to rebase it, which I've now done. Looks like the formatting changes are gone. Thanks for catching this!